### DISCUSSION TITLE
Version 5.0.0 is out!

### DISCUSSION THREAD

**Discussion by ronshnapp** [ronshnapp/MyPTV#33] [Announcements]
A new version of MyPTV is out - 5.0.0

This version includes:
1) a new workflow for the calibration process, using two GUIs - an initial calibration GUI and a final calibration GUI
2) various bug fixes and performance tweaks to the various algorithms

Comments and bug reports will be appreciated!

**Comment by alexlib** [comment]
shall we get back to the discussion about OpenPTV 2.0

**Reply by ronshnapp** [reply]
Yes! Let's build a road map and get going. Let's try to Zoom - perhaps later on today?
